.github/workflows/ci.yaml

```
@@ -89,16 +103,17 @@ jobs:
       - name: Install pnpm
         uses: pnpm/action-setup@v4
         with:
-          version: 10
-      - name: Use Node.js ${{ matrix.node-version }}
+          version: 10.24.0
+          run_install: false
+      - name: Install Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: ${{ matrix.node-version }}
-          cache: yarn
+          node-version: 22
```

```suggestion
          node-version: ${{ matrix.node-version }}
```

Why is it fixed but not dynamic now when `strategy.matrix.node-version` persists?

---

Good catch! Fixed in f2146a2e6.